### PR TITLE
feat(as-3842): add any of following page

### PR DESCRIPTION
**Pull request by demzzydev**
## Ticket Number

https://pins-ds.atlassian.net/browse/AS-3842

## Description of change
Adding any of following page 
Previous PR: https://github.com/Planning-Inspectorate/appeal-planning-decision/pull/1900

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [ ] My commit history in this PR is linear
- [x] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by nssdesikan — CHANGES_REQUESTED** [review]

File: packages/forms-web-app/src/controllers/before-you-start/any-of-following.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    if (errorMessage) {
+      return res.render(VIEW.BEFORE_YOU_START.ANY_OF_FOLLOWING, {
+        errors,
+        errorSummary,
+      });
+    }
+  }
+
+  if (routeUserOption(option) === true) {
+    return res.redirect('/before-you-start/enforcement-notice');
```
[the remark this answers is not in the document]
Comment (demzzydev) [reply packages/forms-web-app/src/controllers/before-you-start/any-of-following.js:52]: What would the benefit be?

File: packages/forms-web-app/src/controllers/before-you-start/any-of-following.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  }
+
+  return res.redirect('/before-you-start/use-a-different-service');
```
[the remark this answers is not in the document]
Comment (demzzydev) [reply packages/forms-web-app/src/controllers/before-you-start/any-of-following.js:55]: Likewise for this?

File: packages/forms-web-app/__tests__/unit/routes/before-you-start/any-of-following.test.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  getAnyOfFollowing,
+} = require('../../../../src/controllers/before-you-start/any-of-following');
+
+describe('routes/before-you-start/any-of-following', () => {
+  beforeEach(() => {
+    // eslint-disable-next-line global-require
+    require('../../../../src/routes/before-you-start/any-of-following');
+  });
+
+  afterEach(() => {
```
[the remark this answers is not in the document]
Comment (demzzydev) [reply packages/forms-web-app/__tests__/unit/routes/before-you-start/any-of-following.test.js]: resolved

File: packages/forms-web-app/src/validators/before-you-start/any-of-following.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+];
+
+const rules = () => {
+  return [
+    body('option')
+      .custom((value, { req }) => {
+        const { option } = req.body;
+
+        if (typeof option === 'string' && !anyOfFollowingOptions.includes(option)) {
+          return true;
```
[the remark this answers is not in the document]
Comment (demzzydev) [reply packages/forms-web-app/src/validators/before-you-start/any-of-following.js]: resolved

File: packages/forms-web-app/src/controllers/before-you-start/any-of-following.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
+/* eslint-disable consistent-return */
+const { VIEW } = require('../../lib/views');
+
+const routingOptions = (option) => {
+  switch (option) {
```
[the remark this answers is not in the document]
Comment (demzzydev) [reply packages/forms-web-app/src/controllers/before-you-start/any-of-following.js]: resolved

**Review by swdpcomputing — DISMISSED** [review]

**Review by swdpcomputing — APPROVED** [review]

**Pull request merged**